client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -41,6 +41,8 @@ object CapiModelEnrichment {
 
   val isGallery: ContentFilter = tagExistsWithId("type/gallery")
 
+  val isPrintShop: ContentFilter = content => !isPictureContent(content) && tagExistsWithId("artanddesign/series/guardian-print-shop")(content)
```

Comment:
not quite what @mxdvl had in mind, I think the order matters in the list below, so we can just move print shop down without adding the extra !isPictureContent logic - I think that's preferable to try and keep things simple? but perhaps it's worth adding some kind of "order matters" comment

Reply:
Personally, I think relying on ordering feels a little brittle, being explicit about excluding pictures from this design type does not greatly add to the complexity and makes our intention clear

Reply:
@rebecca-thompson that seems a lot less brittle 👏!

There was a more fundamental question of why `PrintShopDesign` exists in the first place, which I think is worth exploring and possibly removing altogether.